tests/hikari/impl/test_entity_factory.py

```diff
@@ -6484,6 +6484,7 @@ def template_payload(self, guild_text_channel_payload, user_payload):
                 "system_channel_id": "8",
                 "system_channel_flags": 0,
             },
+            "app": mock_app,
```

Comment:
The app is not deserialized from the payload, so this is not needed

Reply:
Done <3